packages/ponybench/pony_bench.pony

```diff
@@ -114,14 +114,20 @@ actor PonyBench
 
   be _result(name: String, ops: U64, nspo: U64) =>
     _remove(name)
-    let sl = [name, "\t", Format.int[U64](ops where width=10), "\t",
-      Format.int[U64](nspo where width=10), " ns/op"]
+    let sl =
+      [ name
+        "\t"
+        Format.int[U64](ops where width=10)
+        "\t"
+        Format.int[U64](nspo where width=10)
+        " ns/op"
+      ]
```

Comment:
Quick style point to think about and discuss.

How would you feel about using the following bracket style for multiline arrays in the standard library?

```pony
let sl = [
  name
  "\t"
  Format.int[U64](ops where width=10)
  "\t"
  Format.int[U64](nspo where width=10)
  " ns/op"
]
```

Quite subjective, I know, but it feels more balanced to me, and also more consistent with the style being used for lambda curly braces so far.